components/admin_console/admin_definition.jsx

```diff
@@ -188,7 +188,9 @@ export const it = {
         }
         return cloud.subscription.is_paid_tier === 'true';
     },
-    userHasReadPermissionOnResource: (key) => (config, state, license, enterpriseReady, consoleAccess) => consoleAccess?.read?.[key],
+    userHasReadPermissionOnResource: (key) => (config, state, license, enterpriseReady, consoleAccess) => {
+        return consoleAccess?.read?.[key];
+    },
```

Comment:
I believe this change can be reverted. I think keeping it as a one liner fits better with the rest.

Reply:
Have no idea how this happened 🧙 ... Reverting now.